torch/_inductor/decomposition.py

```diff
@@ -522,7 +522,7 @@ def fmax(self: torch.Tensor, other: torch.Tensor) -> torch.Tensor:
     return torch.where(torch.isnan(other) | (other < self), self, other)
 
 
-@register_decomposition(aten.amax)
+@register_decomposition([aten.amax])
```

Comment:
Are you requiring the input to register_decomposition be a list now? It was nice for it to not be a list

Reply:
@zou3519 That was what the existing typing for `register_decomposition` claimed. If we want to relax that, that's OK as well!

Reply:
Let's relax the typing for this